Makefile.toml

```diff
@@ -12,28 +12,33 @@ BUILDSYS_TIMESTAMP = { script = ["date +%s"] }
 BUILDSYS_VERSION_BUILD = { script = ["git describe --always --dirty --exclude '*' || echo 00000000"] }
 BUILDSYS_VERSION_IMAGE = { script = ["awk -F '[ =\"]+' '$1 == \"version\" {print $2}' Release.toml"] }
 BUILDSYS_VARIANT = "aws-k8s-1.15"
+
 # Disallow pulling directly Upstream URLs when lookaside cache results in MISSes as a fallback.
 # To use the upstream source as fallback, override this on the command line and set it to 'true'
 BUILDSYS_UPSTREAM_SOURCE_FALLBACK = "false"
+
 # We require license checks to pass to build an image.  If you're working on a
 # local change and don't have license information yet, you can run with `-e
 # BUILDSYS_ALLOW_FAILED_LICENSE_CHECK=true` to allow the build to continue even
 # if the license check fails.
 BUILDSYS_ALLOW_FAILED_LICENSE_CHECK = "false"
 
+# This controls how many `docker build` commands we'll invoke at once.
+BUILDSYS_JOBS = "8"
```

Comment:
Is this number respected in downstream builds (eg: `make -j1` for each respective process)?

Reply:
It's not - they are somewhat orthogonal concepts. The kernel build can still make use of all cores and we want to keep that property even if we only have a few builds going at once.

Reply:
I see. I read this as though the parameter were prefixed with `BUILDSYS_` where `BUILDSYS_JOBS` meant `JOBS` (as in `-j`) which I'm used to seeing respected up and down the build "call stack".